android/hybridcommon/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -1359,6 +1362,10 @@ fun stubStoreProduct(
             offeringId,
         )
     }
+
+    override fun copyWithPresentedOfferingContext(presentedOfferingContext: PresentedOfferingContext?): StoreProduct {
+        TODO("Not yet implemented")
```

Comment:
Hmm I see `copyWithOfferingId` is actually implemented... Should we copy the implementation here, and update that one to use this one?